web/src/screens/ExecScreen.tsx

```diff
@@ -1,9 +1,37 @@
-function ExecScreen() {
+import React from 'react';
+
+interface Exec {
```

Comment:
in the future, we would keep all these interfaces in a different file (under types.ts or something) just so that they all belong in one place. Just for the future!